tests/integration/test_array_creation.py

```diff
@@ -214,13 +214,27 @@ def test_arange_with_dtype(args, dtype):
     assert x.dtype == y.dtype
 
 
+ARANGE_ARGS_STEP_ZERO = [
+    (0, 0, 0),
+    (0, 10, 0),
+    (-10, 10, 0),
+    (1, 10, 0),
+    (10, -10, 0),
+    (0.0, 0.0, 0.0),
+    (0.0, 10.0, 0.0),
+    (-10.0, 10.0, 0.0),
+    (1.0, 10.0, 0.0),
+    (10.0, -10.0, 0.0),
+]
+
+
 class TestArrangeErrors:
     def test_negative_sizes(self):
         with pytest.raises(ValueError):
-            ###np.arange(-10) returns [] successfully
+            # np.arange(-10) returns [] successfully
```

Comment:
@magnatelee Is this difference in results between numpy and cunumeric here expected or a bug?

Reply:
note that in case there are currently expected failures, we can still have the test run (and avoid leaving commented code in the repo) by using `@pytest.mark.xfail`

Reply:
I like this approach. If an `xfail` test switches to succeeding (when we fix the corresponding bug), will that show up as a test suite failure?

Reply:
@manopapad yes it will which is a nice way to know in case you've accidentally fixed a bug :)

Reply:
@bryevdv I like this approach, too. For AssertionError raised by cuNumeric, do we need to expect ValueError as NumPy in  our test code and make it xfail?

Reply:
>  do we need to expect ValueError as NumPy in our test code and make it xfail?

That is certainly an option although if this is a very simple fix I'd say just fix it (to raise ValueError) in this PR if possible.

Reply:
AssertionError is fixed. It raises ValueError now. Other failures are added with xfail.